lib/ProMotion/styling/styling.rb

```diff
@@ -25,8 +25,10 @@ def set_attribute(element, k, v)
       elsif k.to_s.include?("_") # Snake case?
         set_attribute(element, camelize(k), v)
       else # Warn
-        PM.logger.debug "set_attribute: #{element.inspect} does not respond to #{k}=."
-        PM.logger.log("BACKTRACE", caller(0).join("\n"), :default) if PM.logger.level == :verbose
+        mp "set_attribute: #{element.inspect} does not respond to #{k}=.", force_color: :purple
+        # TODO - remove now, or when fully deprecated - there will be no verbose
+        # check when logger is removed
+        mp "BACKTRACE", caller(0).join("\n") if PM.logger.level == :verbose
```

Comment:
@jamonholmgren what do you want to do here? should I just remove this now since its going to be deprecated and this is something internal?  Or leave it this way until we officially deprecate.